Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.15 as builder
+WORKDIR /go/src/github.com/gojekfarm/stevedore
+COPY ./ ./
+RUN make compile
+
+FROM alpine:latest
+ENV HELM_VERSION="v3.4.0"
+RUN wget -q https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz -O - | tar -xzO linux-amd64/helm > /usr/local/bin/helm
+RUN chmod +x /usr/local/bin/helm
```

Comment:
Shall we combine these?

Reply:
can we have the helm repository file in the repo and copy it to the path rather than adding helm binary and performing helm repo add followed by helm repo update?

Reply:
We can try that... I tried it and failed... So added the helm itself... may be I should once again try now

Reply:
Changed to one line for now... We can try to remove in another PR